packages/ssh-agent/ssh-agent.0.1.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+maintainer: "Reynir Björnsson <reynir@reynir.dk>"
+author: "Reynir Björnsson <reynir@reynir.dk>"
+dev-repo: "https://github.com/reynir/ocaml-ssh-agent.git"
+homepage: "https://github.com/reynir/ocaml-ssh-agent/"
+bug-reports: "https://github.com/reynir/ocaml-ssh-agent/issues/"
+license: "BSD-2-clause"
+build: ["jbuilder" "build" "-p" name]
+depends: [
+  "ocamlfind" {build}
+  "jbuilder" {build}
+  "ppx_cstruct" {build}
+  "ppx_sexp_conv" {build}
```

Comment:
since v0.11.0, `ppx_sexp_conv` introduced a runtime module to depend on, I suspect the `build` here is wrong.